internal/csi-addons/networkfence/fencing.go

```diff
@@ -93,6 +98,16 @@ func (nf *NetworkFence) addCephBlocklist(ctx context.Context, ip string) error {
 func (nf *NetworkFence) AddNetworkFence(ctx context.Context) error {
 	// for each CIDR block, convert it into a range of IPs so as to perform blocklisting operation.
 	for _, cidr := range nf.Cidr {
+		// try range blocklist cmd, if invalid fallback to
+		// iterating through IP range.
+		err := nf.addCephBlocklist(ctx, cidr, true)
```

Comment:
Can you move this to a helper and use a `NetworkFence.hasBlocklistRange` bool? No need to pass it around that way, and it will only be tried once for the `cidr` in `nf.Cidr`.

Reply:
made modifications so we only check fail with invalid command in the first iteration.